tests/integration/components/ember-table-test.js

```diff
@@ -314,3 +320,68 @@ test('Custom footer', async function(assert) {
     );
   }
 });
+
+test('Footer event', async function(assert) {
+  assert.expect(1);
+
+  let callback = () => {
+    assert.ok(true, 'Footer event is sent to outer controller');
+  };
+
+  let columnCount = 10;
+  let columns = generateColumns(columnCount);
+  let footerRows = generateCustomFooterRows(columns);
+
+  this.set('columns', columns);
+  this.set('rows', generateRows(10, columnCount));
+  this.set('footerRows', footerRows);
+
+  if (Ember.VERSION >= '1.13') { // eslint-disable-line no-restricted-globals, no-undef
```

Comment:
`Ember.VERSION` is a string. This comparison works right now because `1.13` was the last of the 1.* cycle, so all versions in 2.* _are_ bigger, string-wise. It would fail when `2.13 >= 2.8` for instance. So, we should always use a semver library, or `ember-compatibility-helpers`